lib/src/command/global_run.dart

```diff
@@ -30,8 +30,10 @@ class GlobalRunCommand extends PubCommand {
     argParser.addFlag('checked', abbr: 'c', hide: true);
     argParser.addMultiOption('enable-experiment',
         help: 'Runs the executable in a VM with the given experiments enabled. '
-            '(Will disable snapshotting, resulting in slower startup)',
+            '(Will disable snapshotting, resulting in slower startup).',
         valueHelp: 'experiment');
+    argParser.addOption('sound-null-safety',
```

Comment:
I think it supposed to be a flag... such that it has a `--sound-null-safety` flag, and a `--no-sound-null-safety` flag.

Reply:
Of course...